code/modules/holiday/halloween/jacqueen.dm

```diff
@@ -0,0 +1,442 @@
+		candies = shuffle(candies)//Shake those ghosts up!
+		for(var/mob/dead/observer/C2 in candies)
+			if(C2.key && C2)
+				key = C2.key
+				message_admins("[C2]/[C2.key] has agreed to go on a date with [C] as Jacqueline.")
+				log_game("HALLOWEEN: [C2]/[C2.key] has agreed to go on a date with [C] as Jacqueline")
+				to_chat(src, "<span class='big spooky'>You are Jacqueline the great pumpqueen, witch Extraordinaire! You're a very Scottish lass with a kind heart, but also a little crazy. You also blew up the wizarding school and you're suspended for a while, so you visited the station before heading home. On your head lies the prize pumpkin of your Mother's pumpkin patch. You're currently on a date with [C] and well, I didn't think anyone would get this far. <i> Please be good so I can do events like this in the future. </i> </span>")
+				return
+			else
+				candies =- C2
+	visible_message("<b>[src]</b> looks to the side, <span class='spooky'>\"Look, Ah like ye but, Ah don't think Ah can right now. If ye can't tell, the stations covered in volatile candies, I've a few other laddies and lassies running after me treats, and tae top it all off, I've the gods breathing down me neck, watching every treat Ah make fer the lot of yous.\" she sighs, \"But that's not a no, right? That's.. just a nae right noo.\"</span>")
+	sleep(20)
+	visible_message("<b>[src]</b> takes off the pumpkin on her head, a rich blush on her cheeks. She leans over planting a kiss upon your forehead quickly befere popping the pumpkin back on her head.")
+	sleep(10)
+	visible_message("<b>[src]</b> waves their arms around, <span class='spooky'>\"There, that aught tae be worth a candy.\"</span>")
+	sleep(20)
+	poof()
+
+/obj/item/clothing/head/hardhat/pumpkinhead/jaqc
+	name = "Jacq o' latern"
+	desc = "A jacqueline o' lantern! You can't seem to get rid of it."
+	icon_state = "hardhat0_pumpkin_j"
+	item_state = "hardhat0_pumpkin_j"
+	item_color = "pumpkin_j"
+	brightness_on = 4
+
+/obj/item/clothing/head/hardhat/pumpkinhead/jaqc/Initialize()
+	. = ..()
+	ADD_TRAIT(src, TRAIT_NODROP, GLUED_ITEM_TRAIT)
+
+/obj/item/clothing/suit/ghost_sheet/sticky
+
+/obj/item/clothing/suit/ghost_sheet/sticky/Initialize()
+	. = ..()
+	ADD_TRAIT(src, TRAIT_NODROP, GLUED_ITEM_TRAIT)
+
+/obj/item/clothing/suit/ghost_sheet/sticky/attack_hand(mob/user)
+	if(iscarbon(user))
+		to_chat(user, "<span class='spooky'><i>Boooooo~!</i></span>")
+		return
+	else
+		..()
+
+/obj/item/clothing/suit/ghost_sheet/sticky/attack_hand(mob/user)
+	if(iscarbon(user))
+		to_chat(user, "<span class='spooky'><i>Boooooo~!</i></span>")
+		return
+	else
+		..()
+
+/datum/reagent/mutationtoxin/pumpkinhead
+	name = "Pumpkin head mutation toxin"
+	id = "pumpkinmutationtoxin"
+	race = /datum/species/dullahan/pumpkin
+	mutationtext = "<span class='spooky'>The pain subsides. You feel your head roll off your shoulders... and you smell pumpkin."
+	//I couldn't get the replace head sprite with a pumpkin to work so, it is what it is.
```

Comment:
You could make a subtype species of dullahan to set the head to pumpkin manually!

Reply:
I did do that!
See: /datum/species/dullahan/pumpkin !